Tests/test_GenBank_unittest.py

```diff
@@ -294,15 +295,16 @@ def test_long_names(self):
         """Various GenBank names which push the column based LOCUS line."""
         original = SeqIO.read("GenBank/iro.gb", "gb")
         self.assertEqual(len(original), 1326)
+        # Acceptability of LOCUS line with length > 80 invalidates some of these tests
         for name, seq_len, ok in [
                 ("short", 1, True),
                 ("max_length_of_16", 1000, True),
                 ("overly_long_at_17", 1000, True),
                 ("excessively_long_at_22", 99999, True),
-                ("excessively_long_at_22", 100000, False),
+                # ("excessively_long_at_22", 100000, False),
                 ("pushing_the_limits_at_24", 999, True),
-                ("pushing_the_limits_at_24", 1000, False),
-                ("longest_possible_len_of_26", 10, False),  # 2 digits
+                # ("pushing_the_limits_at_24", 1000, False),
+                # ("longest_possible_len_of_26", 10, False),  # 2 digits
```

Comment:
Again, rather than commenting out, can you change ``False`` to ``True``?

Also, any ideas for renaming the 26 letter long ID of ``"longest_possible_len_of_26"`` (since this is no longer the maximum possible)? How about ``"old_max_name_length_was_26"`` which is also 26 characters?